core/src/main/scala/com/softwaremill/session/SessionDirectives.scala

```diff
@@ -107,8 +118,8 @@ object SessionOptions {
 trait OneOffSessionDirectives {
   private[session] def setOneOffSession[T](sc: SessionContinuity[T], st: SetSessionTransport, v: T): Directive0 =
     st match {
-      case CookieST => setCookie(sc.clientSessionManager.createCookie(v))
-      case HeaderST => respondWithHeader(sc.clientSessionManager.createHeader(v))
+      case CookieST => respondWithDefaultCookie(sc.clientSessionManager.createCookie(v))
```

Comment:
maybe add a comment why we are using the default version - that the cookie/header might have already been set because of refreshable sessions?